tests/suite/test_batch_startup_times.py

```diff
@@ -503,3 +507,88 @@ def test_ap_waf_policy_vs_batch_start(
         delete_policy(kube_apis.custom_objects, "waf-policy", test_namespace)
 
         assert num is None
+
+
+##############################################################################################################
+
+VSR_COUNT=500
+
+@pytest.fixture(scope="class")
+def vs_vsr_setup(
```

Comment:
this function doesn't remove any created resources because k8s will remove them when the test_namespace is deleted.

Reply:
`test_namespace` fixture is class scoped not method so if there are other tests in same class then those resources will remain until all the tests are executed.